src/main/java/gov/nasa/cumulus/metadata/aggregator/UMMUtils.java

```diff
@@ -297,5 +297,12 @@ public static boolean isNullOrEmpty(String str) {
     public static boolean notNullOrEmpty(String str) {
         return !isNullOrEmpty(str);
     }
+
+    public static Double longitudeTypeNormalizer(Double value) {
+        value = value < -180 ? -180 : value;
```

Comment:
I don't think we need this line -- we really need to account for the case where the longitude is in 0-360 format, which it will only do when longitude > 180. If longitude is < -180, I think the value would be invalid and we should not normalize.